src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -1877,6 +1877,18 @@ public static Properties allProperties() {
         return new Properties(properties);
     }
 
+    public static SortedMap<String, String> getDefaultValues() {
+        SortedMap<String, String> defaultValueMap = new TreeMap<>();
+
+        for (Property property : allProperties().getProperties()) {
+            if (property.describeDefault() != null) {
+                defaultValueMap.put(property.getKey(), property.describeDefault());
+            }
+        }
+
+        return defaultValueMap;
+    }
+
```

Comment:
This section would give us all the default values and decisions which Detect takes during run time for all the properties. Example is "detect.accuracy.required=ALL, detect.timeout=300"